schematic/models/validate_attribute.py

```diff
@@ -421,3 +505,66 @@ def url_validation(self, val_rule: str, manifest_col: str) -> List[List[str]]:
                                 )
                             )
         return errors
+
+    def cross_validation(
+        self, val_rule: str, manifest_col: pd.core.series.Series
+    ) -> List[List[str]]:
+        errors = []
+        missing_values = {}
+        missing_manifest_log={}
+        present_manifest_log=[]
+
+        #parse sources and targets
+        [source_component, source_attribute] = val_rule.split(" ")[1].split(".")
+        [target_component, target_attribute] = val_rule.split(" ")[2].split(".")
+
+        #Get IDs of manifests with target component
+        syn, target_IDs=ValidateAttribute.get_target_manifests(target_component)
+
+        #Read each manifest
+        for target_manifest_ID in target_IDs:
+            entity = syn.get(target_manifest_ID)
```

Comment:
Let's add syn.get wrapper in SynapseStore. We can name it getAsset() and pass the manifest ID. That way we can decouple a bit from Synapse in this validation module.